server.js

```diff
@@ -21,7 +14,23 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send("Netflix data!");
+});
+
+app.get('/movies', (req, res) => {
+  res.status(200).json(netflixData);
+});
+
+app.get("/movies/id/:id", (req,res) => {
+  const { id } = req.params
+  const showId = netflixData.find(show => show.show_id === +id)
+
+  if (!showId) {
+    res.status(404).send("No movie found")
+  } else {
+    res.json(showId)
+  }
```

Comment:
Good that you have a 404 error handling with a specific error message - leaving the user no doubts that there wasn't a movie with that id.